src/request.rs

```diff
@@ -3,6 +3,8 @@
 use std::borrow::Borrow;
 use std::borrow::Cow;
 use std::convert::From;
+#[cfg(all(feature = "tls", unix))]
```

Comment:
Does darwin not count as a unix in this case?

Reply:
https://github.com/sfackler/rust-native-tls/blob/master/Cargo.toml#L23

Reply:
Indeed! Fixed.